src/ProductConstructionService/ProductConstructionService.Api/Dockerfile

```diff
@@ -13,14 +14,18 @@ COPY ["Directory.Packages.props", "Directory.Build.props", "Directory.Build.targ
 COPY ["eng/Versions.props", "./eng/"]
 COPY ["src/ProductConstructionService/ProductConstructionService.Api/ProductConstructionService.Api.csproj", "./ProductConstructionService/ProductConstructionService.Api/"]
 COPY ["src/ProductConstructionService/ProductConstructionService.ServiceDefaults/ProductConstructionService.ServiceDefaults.csproj", "./ProductConstructionService/ProductConstructionService.ServiceDefaults/"]
+COPY ["src/Maestro/Maestro.AzureDevOps/Maestro.AzureDevOps.csproj", "./Maestro/Maestro.AzureDevOps/"]
```

Comment:
Can we sort these alphabetically? Or are they sorted by how frequently we expect those to change?

Reply:
sorted, improved indentation so it's readable